packages/rrweb/src/record/index.ts

```diff
@@ -744,7 +744,7 @@ interface PrivateGetCanvasManagerOptions
   extends PublicGetCanvasManagerOptions,
     Pick<CanvasManagerConstructorOptions, PrivateOptions> {}
 
-function _getCanvasManager(
+export function _getCanvasManager(
```

Comment:
this function is used in sentry-javascript since the mirror and window is needed in CanvasManager for manual snapshots to work

Reply:
hmm this seems smelly to me 😅 if this is needed, we should maybe just export this as `getCanvasManager` and use this directly? not sure if that has problems... but we should avoid exporting a `_XXX` method I'd say!